browser/model/jdk-install.js

```diff
@@ -34,7 +35,7 @@ class JdkInstall extends InstallableItem {
   }
 
   detectExistingInstall() {
-    let versionRegex = /version\s"(\d+\.\d+\.\d+)_.*"/;
+    let versionRegex = /version\s"(\d+\.\d+\.\d+).*"/;
```

Comment:
This is not going to match the new java version string. For java 9+ the output for `java -version` is
`java version "$majorVersion" yyyy-mm-dd`, `$majorVersion` being the 9+

Also, if you want to compare `$majorVersion` to say `1.8.0` it doesn't work straight away, but semver seems to have some API to adjust it.

Reply:
For windows release output is:
```shell
c:\p\rh\dpi>java -version
openjdk version "9.0.4.1-redhat"
OpenJDK Runtime Environment (build 9.0.4.1-redhat+11)
OpenJDK 64-Bit Server VM (build 9.0.4.1-redhat+11, mixed mode)
```
and installer would show

![image](https://user-images.githubusercontent.com/620330/38747364-870310a6-3eff-11e8-8c88-37546dfad316.png)

@mohitsuman is checking macOS output right now.

Reply:
On mac it is:

``` shell
openjdk version "9.0.4"
OpenJDK Runtime Environment (build 9.0.4+11)
OpenJDK 64-Bit Server VM (build 9.0.4+11, mixed mode)
```
Installer shows messages below:

![image](https://user-images.githubusercontent.com/620330/38749291-f76c034c-3f05-11e8-8aa0-10a1c0acb6ef.png)

Reply:
@jrichter1 so my guess you are testing on RHEL, aren't you?

Reply:
yeah for macOS 10.13.4 , it shows:

```
dhcp193-9:~ mohitsuman$ java -version
java version "9.0.4"
Java(TM) SE Runtime Environment (build 9.0.4+11)
Java HotSpot(TM) 64-Bit Server VM (build 9.0.4+11, mixed mode)